main.go

```diff
@@ -193,10 +202,16 @@ var cmdRevert = cli.Command{
 	Name:      "revert",
 	Usage:     "Revert to previously installed version of ipfs.",
 	ArgsUsage: " ",
-	Description: `'revert' will check if a previous update left a stashed
+	Description: `
+   'revert' will check if a previous update left a stashed
    binary and overwrite the current ipfs binary with it.
+
+   Using 'revert' will not run any datastore migrations. For that, use
+   'ipfs-update install --downgrade <prev-version>'.
```

Comment:
Hm. I'd _expect_ revert to _always_ run migrations. I'd expect it to be a "undo what I just did" command.

Reply:
Yeah, I did not do that, but I'd expect the same and thought about warning about it.

On the other hand, it is good to have a way to downgrade without running migrations, but this is not the best. We should fix this on a separate PR though.

Reply:
https://github.com/ipfs/ipfs-update/issues/127